qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
+  Given form "Send CV" with blank fields
+  When I enter valid data in the field "Name"
+  And I enter valid data in the field "Email"
```

Comment:
Usually these steps are written from the user's point of view. Maybe it might be smth like that: user enters valid data in the field "Name".

Reply:
done